### PR TITLE
added CsvReader and tests

### PR DESCRIPTION
**Pull request by Rudiachenko**

### CODE REVIEW

File: src/main/java/amazonreviewers/project/service/CsvFileReaderImpl.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    @Override
+    public List<List<String>> readFile(String csvFilePath) {
+        String[] line;
+        List<List<String>> data = new ArrayList<>();
+        try (CSVReader reader = new CSVReader(new FileReader(csvFilePath))) {
+            while ((line = reader.readNext()) != null) {
+                data.add(Arrays.stream(line)
+                        .collect(Collectors.toList()));
+            }
```
Comment (andromash) [inline src/main/java/amazonreviewers/project/service/CsvFileReaderImpl.java:21]: You can do this logic with standard libraries

File: src/main/java/amazonreviewers/project/service/CsvFileReaderImpl.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        }
+        if (data.isEmpty()) {
+            throw new NoSuchElementException("The file is empty.");
+        }
+        StringJoiner header = new StringJoiner(",");
+        for (String element : data.get(0)) {
+            header.add(element);
+        }
+        if (!header.toString().equals(HEADER)) {
+            throw new NoHeaderException("File without header.");
```
Comment (andromash) [inline src/main/java/amazonreviewers/project/service/CsvFileReaderImpl.java]: CSV parsing is another homework, so exception should be in another class. The task here - simply read the file.

**Review by andromash — COMMENTED** [review]
Nice!

File: src/main/java/amazonreviewers/project/service/CsvFileReaderImpl.java (hunk 2 above)
Comment (dmytro-hryhoruk) [reply src/main/java/amazonreviewers/project/service/CsvFileReaderImpl.java]: agree, and I recommend you to use existing libraries too. It's ease to divide headers with comma, but you text field(for example) may contain comma by itself.

File: src/main/java/amazonreviewers/project/service/CsvFileReaderImpl.java (hunk 1 above)
Comment (dmytro-hryhoruk) [reply src/main/java/amazonreviewers/project/service/CsvFileReaderImpl.java:21]: agree

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**